app/src/main/java/com/example/socialapp/network/Api.java

```diff
@@ -10,8 +11,12 @@ public interface Api {
     @GET("oauth/access_token")
     Call<AccessTokenResponse> getAccessToken(@Query("code") String code,
                                              @Query("client_id") String clientId,
-                                       @Query("redirect_uri") String redirectUri,
+                                             @Query("redirect_uri") String redirectUri,
                                              @Query("client_secret") String client_secret);
     @GET("me/posts")
-    Call<Posts> getPosts(@Query("fields") String fields,@Query("access_token") String accessToken);
+    Call<Posts> getPosts(@Query("fields") String fields, @Query("access_token") String accessToken);
+
+   // Address getAddress(int customerId, String customername)
```

Comment:
We can remove commented code if it's not required.

Reply:
Done.